applications/tari_console_wallet/src/cli.rs

```diff
@@ -129,7 +129,10 @@ pub enum CliCommands {
     ClearCustomBaseNode,
     InitShaAtomicSwap(SendTariArgs),
     FinaliseShaAtomicSwap(FinaliseShaAtomicSwapArgs),
-    ClaimShaAtomicSwapRefund(ClaimShaAtomicSwapRefundArgs),
+    ClaimShaAtomicSwapRefund(ClaimAtomicSwapRefundArgs),
+    InitBlake2AtomicSwap(SendTariArgs),
+    FinaliseBlake2AtomicSwap(FinaliseBlake2AtomicSwapArgs),
+    ClaimBlake2AtomicSwapRefund(ClaimAtomicSwapRefundArgs),
```

Comment:
Is there any reason why we dont change `ClaimShaAtomicSwapRefund` -> `ClaimAtomicSwapRefund` and  the same for the other 2 functions?
Then we just add an enum for the hash algo?

We are duplicating a lot of code just for a simple hash change?

Reply:
Yeah, I wanted to do this. But, I didn't want to change the already existing interfaces without further comments.

Reply:
Since, we currently have two functions for sending and claiming atomic swap txs, we need to distinguish these in the enum. A possibility is to integrate everything into a single function, with a hasher as a generic parameter. Not sure, what is preferable.